jaeger-zipkin/build.gradle

```diff
@@ -1,18 +1,15 @@
 description = 'Integration library for Zipkin'
 
 dependencies {
-    compile group: 'io.zipkin.reporter', name: 'zipkin-reporter', version: '1.1.2'
-    compile group: 'io.zipkin.reporter', name: 'zipkin-sender-urlconnection', version: '1.1.2'
+    implementation "io.zipkin.reporter2:zipkin-reporter-bom:2.7.5"
 
-    compile group: 'io.zipkin.java', name: 'zipkin', version: '2.8.1'
-    compile group: 'io.zipkin.reporter2', name: 'zipkin-reporter', version: '2.6.0'
+    implementation 'io.zipkin.reporter2:zipkin-reporter'
+    implementation 'io.zipkin.reporter2:zipkin-sender-urlconnection'
```

Comment:
This is new to me, it's related to https://github.com/jaegertracing/jaeger-client-java/pull/456/files#diff-88b7c47e47b8ee65263b6784b86fa0a7R1

I got this when I included it into a different project

> [WARNING] The POM for io.jaegertracing:jaeger-zipkin:jar:0.29.1-SNAPSHOT is invalid, transitive dependencies (if any) will not be available, enable debug logging for more details


https://docs.gradle.org/4.6/userguide/java_library_plugin.html#sec:java_library_separation

> Dependencies appearing in the api configurations will be transitively exposed to consumers of the library, and as such will appear on the compile classpath of consumers. Dependencies found in the implementation configuration will, on the other hand, not be exposed to consumers, and therefore not leak into the consumers' compile classpath. This comes with several benefits:

We basically require our clients to explicitly add dependency on `zipkin-sender-urlconnection` to make this work. If am correct it will fail on `java.lang.NoClassDefFoundError: zipkin2/reporter/urlconnection/URLConnectionSender` even if you use different sender.